apps/api/src/server/migration/migrators/nsip-representation-migrator.js

```diff
@@ -43,10 +45,20 @@ export const migrateRepresentations = async (representations) => {
 			}
 		}
 
-		// TODO attachments
-		// for (const attachmentId in representationEntity.attachmentIds) {
-		// 	// create RepresentationAttachment, Document
-		// }
+		await Promise.all(
+			representation.attachmentIds.map(async (attachmentId) => {
+				return documentRepository.getById(attachmentId).then((documentRow) => {
+					if (!documentRow || !documentRow.guid) {
+						return Promise.reject(new Error(`Failed to get document with ID ${attachmentId}`));
+					}
+
+					return representationAttachmentRepository.addApplicationRepresentationAttachment(
```

Comment:
we may need to use an "upsert" type function here, so if we are running this function for a subsequent time, we don't create new RepresentationAttachment rows that have already been created by a previous run

Reply:
Sounds good. Addressed!